.cbmc-batch/include/openssl/crypto.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
```

Comment:
These are included by the SDK code, but there are no functions declared there needed here.  Why not just use the SDK headers?  Because the CI was failing this proof because the CI was linking against the OpenSSL implementation and not our models.  Data structures in OpenSSL definitions were clashing with our model abstractions of the data structures.  This has to be a bug in CI.

Reply:
Comment explaining what this file is.